src/utils/processLinkData.js

```diff
@@ -0,0 +1,18 @@
+import { format } from "date-fns";
+import processTimeData from "./processTimeData";
+
+//시간 형식 바꿔주기 위해!!
+const processLinkData = (link) => {
+  const { id, createdAt, url, imageSource, description } = link;
+  return {
+    id,
+    url,
+    imageSource,
+    alt: "대표 이미지",
```

Comment:
P3:
만약 alt 속성을 추가해주실거면 더 의미있는 text면 좋겠네요.

```suggestion
    alt: `${url} 대표 이미지`,
```